jaxrs/WFLY-12298_Change_RESTEASY_settings.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-12298 Support configuring RESTEasy with CLI
```

Comment:
Regarding terminology it would probably be better to loose the reference to the CLI, the CLI is just one of many potential management clients.

As I understand it this RFE is specifically about exposing RESTEasy configuration using the application servers management model.

It is true that the admin console may need it's own additional RFE for any console specifics but I don't think that makes this specifically about the CLI.

Reply:
Done. Thanks, @darranl.